docs/source/development/CDFs/cdf_requirements.rst

```diff
@@ -0,0 +1,343 @@
+#####################
+CDF File Requirements
```

Comment:
IMAP-Specific CDF Requirements ?

Do we want to be explicit that these are IMAP specific requirements since you say these are necessary for this mission in the next sentence?